.github/workflows/ci.yaml

```diff
@@ -106,7 +105,10 @@ jobs:
 
   ffi:
     needs: build
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
Add MacOS testing since we are now providing pre-built binaries for both linux and mac on x86+arm